src/components/workspace-utils.js

```diff
@@ -180,6 +182,11 @@ export const SnapshotInfo = ({
       ]),
       div({ style: { paddingLeft: '1rem' } }, [
         div({ style: Style.dashboard.header }, ['Linked Data Repo Snapshot']),
+        h(SnapshotLabeledInfo, {
+          title: 'Data Repo Id:', text: [h(Link, {
+            href: `${getConfig().dataRepoUrlRoot}/snapshots/${snapshotId}`
+          }, [snapshotId]), h(ClipboardButton, { 'aria-label': 'Copy data repo id to clipboard', text: snapshotId, style: { marginLeft: '0.25rem' } })]
+        }),
```

Comment:
could this open in a new window? You should just be able to add `target: '_blank',` to the `Link` definition